genrl/deep/agents/dqn/dqn.py

```diff
@@ -32,29 +32,45 @@
 class DQN:
     """
     Deep Q Networks
-    Paper: (DQN) https://arxiv.org/pdf/1312.5602.pdf
-    Paper: (Double DQN) https://arxiv.org/abs/1509.06461
-    :param self.network_type: (str) The deep neural network layer types
-        ['MLP', 'CNN']
-    :param env: (Gym environment) The environment to learn from
-    :param double_dqn: (boolean) For training Double DQN
-    :param dueling_dqn: (boolean) For training Dueling DQN
-    :param noisy_dqn: (boolean) For using Noisy Q
-    :param categorical_dqn: (boolean) For using Distributional DQN
-    :param parameterized_replay: (boolean) For using a prioritized buffer
-    :param epochs: (int) Number of epochs
-    :param max_iterations_per_epoch: (int) Number of iterations per epoch
-    :param max_ep_len: (int) Maximum steps per episode
-    :param gamma: (float) discount factor
-    :param lr: (float) learing rate for the optimizer
-    :param batch_size: (int) Update batch size
-    :param replay_size: (int) Replay memory size
-    :param tensorboard_log: (str) the log location for tensorboard
-        (if None, no logging)
-    :param seed : (int) seed for torch and gym
-    :param render : (boolean) if environment is to be rendered
-    :param device : (str) device to use for tensor operations; 'cpu' for cpu
-        and 'cuda' for gpu
+    Paper (DQN) https://arxiv.org/pdf/1312.5602.pdf
+    Paper (Double DQN) https://arxiv.org/abs/1509.06461
+
+    :param network_type: The deep neural network layer types ['mlp']
```

Comment:
You can add `'cnn'` also as it's been merged

Reply:
yea sorry, i'll add this